src/styles/components/_related-components.scss

```diff
@@ -45,14 +43,11 @@ $breakpoint-move-arrow-left: 38.125rem; // 610px/16
         filter: invert(1);
       }
 
-      @media (min-width: $breakpoint-move-arrow-left) {
-        right: 85%;
-        width: 2.875rem; // 46px/16
-        height: 3.5625rem; // 57px/16
-      }
-
       @media (min-width: $breakpoint-large-screen-860) {
-        right: 50%;
+        width: 2.8rem; // 60px/16
+        height: 3.4rem; // 75px/16
```

Comment:
Same thing here: 2.8rem = ~45px; 3.4rem = ~54px

Reply:
Or maybe we just delete them? I don't know.

Reply:
I've found them useful when refactoring! I'm make sure to fix these before the merge.

Reply:
I've found them extremely useful while refactoring! Thanks for pointing out the inconsistencies--they should be fixed in 5e756a7.